views/index.handlebars

```diff
@@ -1,109 +1,202 @@
+<!DOCTYPE html>
+<html>
```

Comment:
The <html>, <head>, and <body> tags were all extracted into the main.handlebars layout and it looks like they're back in here. Since you started this before the node rewrite there have been some pretty substantial changes and I'm not really sure how to merge this. I think it would be easier to get this merged if you were to open a new PR with just the social share links. That part all looks good and for a new PR I think you'd just need to update the `bottom_nav.handlebars` partial and `locations-list-map.js` with the HTML / JS you've already written

Reply:
That makes sense